app/controllers/teams_controller.rb

```diff
@@ -38,6 +56,6 @@ def set_team
   end
 
   def team_params
-    params.require(:team).permit(:name)
+    params.require(:team).permit(:name, :description)
```

Comment:
Revert this, since it looks like for update you just want to modify the description, not the name.